config/config.go

```diff
@@ -748,6 +752,7 @@ func NewConfig() (c *Config) {
 				IstiodPollingIntervalSeconds:      20,
 				RootNamespace:                     "istio-system",
 				UrlServiceVersion:                 "",
+				ValidationReconcileInterval:       util.AsPtr(time.Minute),
```

Comment:
Wouldn't it better to accept the ValidationReconcileInterval in seconds, as in milliseconds it takes a big numbers and probably it will not be necessary configure that precise level. Like IstiodPollingIntervalSeconds we have in seconds.

Reply:
I see now, it accepts strings.